task-list.js

```diff
@@ -0,0 +1,29 @@
+let tasks = [{
```

Comment:
Eventually we want to start with empty `tasks` array, so there's no need to keep it in a separate file. Just move the `const tasks = [];` line to the beginning of `generate-task-list.js` file and delete this file.